.github/shared/src/branch.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
I didn't really see a good place to put something `branch` related like this. I DID consider `sdk-types.js`, but that seemed like it was type related specifically so I didn't put it there.